tmva/sofie/src/RModel.cxx

```diff
@@ -597,6 +609,12 @@ void RModel::WriteInitializedTensorsToFile(std::string filename) {
             std::vector<int64_t> tensorDataVector(data , data + length);
             outputDir->WriteObjectAny(&tensorDataVector, "std::vector<int64_t>", tensorName.c_str());
          }
+         else if(item.second.fType == ETensorType::BOOL) {
+            const std::shared_ptr<void> ptr = item.second.fData; // shared_ptr<void> instance
+            const bool* data = (std::static_pointer_cast<bool>(item.second.fData)).get();
+            std::vector<bool> tensorDataVector(data , data + length);
+            outputDir->WriteObjectAny(&tensorDataVector, "std::vector<bool>", tensorName.c_str());
+         }
```

Comment:
Same thing here, we don't really require I guess to write boolean vectors as initialized tensors?